Core/CMP API Specification.md

```diff
@@ -871,7 +773,7 @@ window.__gpp_stub = function ()
  if (cmd === 'ping')
  {
   return {
-  gppVersion      : '1.0', // must be “Version.Subversion”, current: “1.0”
+  gppVersion      : '1.1', // must be “Version.Subversion”, current: “1.1”
```

Comment:
the 3 "new" properties (sectionList, applicableSections, gppString) need to be added to ping, addEventListener and removeEventlisterner:

sectionList:[],
applicableSections:[-1], //or 0 or ID set by publisher
gppString:''

Reply:
updated